climpred/bootstrap.py

```diff
@@ -360,6 +361,84 @@ def varweighted_mean_period_threshold(control, sig=95, bootstrap=500, time_dim='
     )
 
 
+def bootstrap_initialized_skill(
+    hind,
+    verif,
+    iterations,
+    metric='pearson_r',
+    comparison='m2e',
+    alignment='same_verifs',
+    dim='init',
+    resample_dim='member',
+    compute=verify_hindcast,
+    **metric_kwargs,
+):
+    """Bootstraps the initialized forecast metric N times."""
+    # Convert/check cftime
+    init = []
+    # TODO: Change this out to a multi-level indexing strategy to perform at once.
+    for i in range(iterations):
+        hindcast_resampled = _resample(hind, resample_dim)
+        init_skill = compute(
+            hindcast_resampled,
+            verif,
+            metric=metric,
+            comparison=comparison,
+            alignment=alignment,
+            add_attrs=False,
+            dim=dim,
+            **metric_kwargs,
+        )
+        # reset inits when probabilistic, otherwise tests fail
+        if (
+            resample_dim == 'init'
+            and metric.probabilistic
```

Comment:
Isn't metric a string?
'pearsonr'.probabilistic

Reply:
So @aaronspring made `metric` and `comparison` classes a bit back. They all have simple attributes like this that we can track.